client/app/components/stepper/components/header/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+   "ovhManagerComponentStepper_header": "Indicateur de progression"
```

Comment:
It could be discussed but I personally prefer using `snake_case` only when dealing with translation key.

Related to: ovh-ux/ovh-ux-guidelines#6